src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.html

```diff
@@ -104,7 +104,6 @@
             [isBuilding]="false"
             [participation]="participation"
             [exercise]="exercise"
-            [short]="true"
```

Comment:
Why is this change? Or was this overlooked?

Reply:
Ah, seams I committed this in the wrong pr. But I wanted to change this non the less.
Do you think this causes any problems here?

Reply:
no, just wanted to ensure this is an intentional change.